requirements.txt

```diff
@@ -1,17 +1,18 @@
-Sphinx==4.5.0
+Sphinx>=5,<6
```

Comment:
This is the only thing that scares me. I don't see anything obviously broken tho, and if the build doesn't have warnings we're probably fine.

Reply:
Yeah, I'd be pro moving all the way up as well. I actually started looking at that over in https://github.com/writethedocs/www/pull/2246 and abandoned it though.